# HG changeset patch
# User Rik <rik@octave.org>
# Date 1435942535 25200
#      Fri Jul 03 09:55:35 2015 -0700
# Node ID 9de6949ec15f49c16053f63f1d4867789f51a8d0
# Parent  c5a8eff5a05dfd8e2899189cdcac4209f54bccf9
graphics_toolkit.m: Fix typo in error message.

* graphics_toolkit.m: Change 'to' to 'too'.

diff --git a/scripts/plot/util/graphics_toolkit.m b/scripts/plot/util/graphics_toolkit.m
--- a/scripts/plot/util/graphics_toolkit.m
+++ b/scripts/plot/util/graphics_toolkit.m
@@ -82,17 +82,17 @@ function retval = graphics_toolkit (name
   endif
 
   if (! any (strcmp (loaded_graphics_toolkits (), name)))
     ## FIXME: Special gnuplot handling for versions < 4.2.5 (bug #44978).
     ## This can probably be deleted in the future once RHEL upgrades gnuplot.
     if (strcmp (name, "gnuplot"))
       valid_version = __gnuplot_has_feature__ ("minimum_version");
       if (valid_version != 1)
-        error ("graphics_toolkit: gnuplot version to old.");
+        error ("graphics_toolkit: gnuplot version too old.");
       endif
     endif
     feval (["__init_", name, "__"]);
     if (! any (strcmp (loaded_graphics_toolkits (), name)))
       error ("graphics_toolkit: %s toolkit was not correctly loaded", name);
     endif
   endif
 
